music/src/music_rnn/model.ts

```diff
@@ -155,11 +162,12 @@ export class MusicRNN<A extends controls.ControlSignalUserArgs> {
    * @param steps How many steps to continue.
    * @param temperature (Optional) The softmax temperature to use when sampling
    * from the logits. Argmax is used if not provided.
-   * @param controlSignalArgs (Optional) Arguments for creating control tensors.
+   * @param chordProgression (Optional) Chord progression to use as
+   * conditioning.
    */
   async continueSequence(
       sequence: INoteSequence, steps: number, temperature?: number,
-      controlSignalArgs?: A): Promise<INoteSequence> {
+      chordProgression = [constants.NO_CHORD]): Promise<INoteSequence> {
```

Comment:
Is NO_CHORD a valid input to a chord encoder?
I was actually thinking this would be undefined by default, and then we would throw an error if a chordEncoder existed but the user didn't pass in a chordProgression, and vice-versa. This is going to be confusing for people, so we need to make the failures really clear.

Reply:
NO_CHORD is a valid input to a chord encoder.  It's a question of what should happen if chords are not passed to a chord-conditioned model.  I think you're right, it should error.